src/server-conn/fetchUsers.jsx

```diff
@@ -0,0 +1,16 @@
+const fetchUsers = async(pageNo) => {
+    const res = await fetch('https://gorest.co.in/public-api/users?page='+pageNo,{})
+    const data = await res.json()
```

Comment:
is `await` needed for `line 3`

Reply:
yes sir await is needed here.without it `data` variable is not getting any value assigned to it